### PR TITLE
fix: Contributor Insight & Repository Insight uses wrong copy

### LINKED ISSUE #3213: Bug: Delete Contributor Insight uses wrong copy
### Describe the bug

1.  The Delete section uses Workspace instead of Contributor Insight

### Steps to reproduce

1. Go [here](https://app.opensauced.pizza/workspaces/70809527-03ce-41ec-a159-fc664651afdb/contributor-insights/05d267d4-d5e3-4877-8e54-306e6bb35519/edit) or to any edit contrib insight page.

## Fix

Copy should be updated to:

Delete Contributor Insight

Once you delete a Contributor Insight, you're past the point of no return.

### Do you have any images or screen captures?

image

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by Lalithkumarponnambalam**
## Related Tickets & Documents

Fixes #3213

## Mobile & Desktop Screenshots/Recordings

Screenshot from 2024-04-20 21-40-04

Screenshot from 2024-04-20 21-39-31

## Tier (staff will fill in)

- [x] Tier 1
- [x] Tier 2
- [x] Tier 3
- [x] Tier 4

### CODE REVIEW

File: pages/workspaces/[workspaceId]/contributor-insights/[listId]/edit.tsx [hunk 1]
```diff
@@ -187,9 +187,11 @@ export default function ContributorInsightEditPage({
...
-              Delete Workspace
+              Delete Contributor Insight
             </Title>
-            <Text className="my-4">Once you delete a workspace, you&apos;re past the point of no return.</Text>
+            <Text className="my-4">
+              Once you delete a Contributor Insight, you&apos;re past the point of no return.
+            </Text>
 
             <Button onClick={() => setIsDeleteModalOpen(true)} variant="destructive" className="w-fit">
               Delete workspace
```
Comment (BekahHW) [inline pages/workspaces/[workspaceId]/contributor-insights/[listId]/edit.tsx:197]: This should be "Delete Insight"

File: pages/workspaces/[workspaceId]/repository-insights/[insightId]/edit.tsx [hunk 2]
```diff
@@ -194,9 +194,9 @@ export default function RepoInsightEditPage({ insight, workspaceId, isOwner, bea
...
           <div className="flex flex-col p-6 rounded-2xl bg-light-slate-4">
             <Title className="!text-1xl !leading-none !border-light-slate-8 border-b pb-4" level={4}>
-              Delete Insight
+              Delete Repository Insight
             </Title>
-            <Text className="my-4">Once you delete an insight, you&apos;re past the point of no return.</Text>
+            <Text className="my-4">Once you delete a Repository Insight, you&apos;re past the point of no return.</Text>
 
             <Button onClick={() => setIsDeleteModalOpen(true)} variant="destructive" className="w-fit">
               Delete insight
```
Comment (BekahHW) [inline pages/workspaces/[workspaceId]/repository-insights/[insightId]/edit.tsx:202]: Can you capitalize "Insight" here?

**Review by BekahHW — CHANGES_REQUESTED** [review]
Looking good! Just a couple of minor changes and should be ready to merge.

**Review by nickytonline — APPROVED** [review]
Thanks @Lalithkumarponnambalam!

:shipit:

**Pull request merged**